### PR TITLE
Added a note on logging in to the image registry KO_DOCKER_REPO points at

### PR DESCRIPTION
**Pull request by syedriko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,9 @@ customresourcedefinition.apiextensions.k8s.io/warmimages.mattmoor.io configured
 `ko` has four commands, most of which build and publish images as part of
 their execution.  By default, `ko` publishes images to a Docker Registry
 specified via `KO_DOCKER_REPO`.
+You'll need to be authenticated with your `KO_DOCKER_REPO` before pushing
```
Comment (matzew) [inline README.md]: nit:  I'd add a **Note:** and an empty line above this new sentence

Comment (syedriko) [reply README.md]: You got it.

**Comment by ImJasonH** [comment]
Thanks!

**Pull request merged**